src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -116,6 +117,26 @@ public string Name
         }
 
         /// <summary>
+        /// User friendly name, that is used in InCanvasSearch.
+        /// </summary>
+        public string UserFriendlyName
+        {
+            get
+            {
+                if (!String.IsNullOrWhiteSpace(userFriendlyName))
+                    return userFriendlyName;
+                else
+                    return name;
+            }
+            set
```

Comment:
This should be `protected set` if that is possible, I don't see how/why external codes need to set this value.